packages/zmarkdown/webpack.config.js

```diff
@@ -1,3 +1,4 @@
+const webpack = require('webpack');
```

Comment:
We use `eslint` to ensure consistency of the code style, and require no semicolon at the end of lines, please remove it, that should make the tests work.